templates/zerver/help/share-a-message-or-conversation.md

```diff
@@ -0,0 +1,27 @@
+# Sharing a message or conversation
+
+A message or conversation can be shared on Zulip by creating
+ a link to the conversation and sharing the link.
+
+1. To create a link to a conversation, first locate the relevant message.
+
+2. Hover over the message to reveal a {!down-chevron.md!}.
+
+3. Click on the {!down-chevron.md!} and select
+ the **Link to conversation** option from the dropdown.
+
+4. Upon clicking **Link to conversation**, Zulip will scope into just that
```

Comment:
Maybe say "narrow" or "filter" instead of "scope"?  Does the style guide say anything about our preferred term here?